node-runner-cli/release_ymls/radix-fullnode-compose.yml

```diff
@@ -1,55 +1,58 @@
-version: '2.2'
+---
+version: '2.4'
 services:
   core:
-    image: radixdlt/babylon-node:1.3.1
-    init: true
-    ulimits:
-      nofile:
-        soft: "65536"
-        hard: "65536"
-      memlock: "-1"
-    restart: unless-stopped
-    mem_limit: 12000m
     cap_add:
-      - NET_ADMIN
+    - NET_ADMIN
     environment:
-      RADIXDLT_NETWORK_SEEDS_REMOTE: "${RADIXDLT_NETWORK_NODE}"
-      RADIXDLT_LOG_LEVEL: info
-      JAVA_OPTS: --enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m -XX:+HeapDumpOnOutOfMemoryError -XX:+UseCompressedOops -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector
-      RADIXDLT_VALIDATOR_KEY_LOCATION: "/home/radixdlt/node-keystore.ks"
-      RADIXDLT_SIGN_ENABLE: "true"
+      JAVA_OPTS: --enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m
```

Comment:
can we please move env vars to a .env file? it would help to readability

Reply:
Lets add this in the next PR. I will create some tickets for that.